components/compositing/windowing.rs

```diff
@@ -55,6 +54,8 @@ pub enum EmbedderEvent {
     Refresh,
     /// Sent when the window is resized.
     WindowResize,
+    /// Sent when the platform theme changes.
```

Comment:
should platform be changed to device?

Reply:
No, I think `platform` is probably good.